### PR TITLE
[TMVA][SOFIE] Support for parsing BatchNorm in Keras

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds support for parsing Batch Normalization layer in `SOFIE::PyKeras`.

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build

File: tmva/pymva/src/RModelParser_Keras.cxx [hunk 1]
```diff
@@ -481,14 +519,16 @@ RModel Parse(std::string filename){
       // inference code.
       else if(fLayerType == "Dense")
          rmodel.AddBlasRoutines({"Gemm", "Gemv"});
+      else if (fLayerType == "BatchNormalization")
```
Comment (lmoneta) [inline tmva/pymva/src/RModelParser_Keras.cxx:522]: Thank you for adding this! It is needed for the BatchNorm. 
Strange it was working before without these declarations.

File: tmva/CMakeLists.txt [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
+find_package(LAPACK)
```
Comment (lmoneta) [inline tmva/CMakeLists.txt:7]: Why do you need this? 
 We have already 
`find_package(BLAS)` in `cmake/modules/SearchInstalledSoftware.cmake` and it should be enough

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

File: tmva/pymva/src/RModelParser_Keras.cxx (hunk 1 above)
Comment (sanjibansg) [reply tmva/pymva/src/RModelParser_Keras.cxx:522]: I think this was already there in the ONNX Parser, but for supporting this in Keras, I had to add it this way.

File: tmva/CMakeLists.txt (hunk 2 above)
Comment (sanjibansg) [reply tmva/CMakeLists.txt]: Ahh yes, removed it now. Thanks!

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default with flags -Dtmva-sofie=On

**Review by lmoneta — APPROVED** [review]
